### PR TITLE
Update docs for 0.34

### PR DESCRIPTION
**Pull request by mazameli**
This adds:
- Documentation for ability to display data values on line, area, and bar charts
- Updated SQL editor screenshots
- Documentation and examples for MongoDB variables

### CODE REVIEW

File: docs/users-guide/05-visualizing-results.md [hunk 1]
```diff
@@ -189,10 +189,32 @@ By default, Metabase will automatically choose a good way to bin your results. B
...
+**Display**
+
+There's quite a bit you can do on this tab:
+
+- Set the colors and labels for the series on your chart
+- Change the style of your lines for Line and Area charts, and choose whether or not to display dots on them
+- Use the "Replace missing values with…" setting to change how your chart deals with missing values. You can user linear interpolation, or display those points as zero or as nothing.
+- Add a goal line. This can be used in conjunction with [alerts](15-alerts.md) to send an email or a Slack message when your metric cross this line.
+- If you're looking at a time series chart, you can turn on a trend line to show where things are heading.
+- Show values on data points. The default setting will try and fit as many values on your chart as will fit nicely, but you can also force Metabase to show the values for each and every data point, which it will do begrudgingly.
```
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]: `which it will do begrudgingly` 😄

File: docs/users-guide/05-visualizing-results.md [hunk 2]
```diff
@@ -189,10 +189,32 @@ By default, Metabase will automatically choose a good way to bin your results. B
...
-- **Axes** — this is where you can hide axis markers or change their ranges, and turn split axes on or off. You can also configure the way your axes are scaled, if you're into that kind of thing.
-- **Labels** — if you want to hide axis labels or customize them, here's where to go.
+**Data**
+
+Here's where you can choose the columns you want to plot on your x and y axes. This is mostly useful if your table or result set contains more than two columns, like if you're trying to graph fields from an unaggregated table. You can also add additional metrics to your chart by clicking the `Add another series` link below the y-axis dropdown, or break your current metric out by an additional dimension by clicking the `Add a series breakout` link below the x-axis dropdown (note that you can't add an additional series breakout if you have more than one metric/series).
+
+**Display**
+
+There's quite a bit you can do on this tab:
```
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]: In vs on here?
```suggestion
There's quite a bit you can do in this tab:
```

File: docs/users-guide/05-visualizing-results.md [hunk 3]
```diff
@@ -189,10 +189,32 @@ By default, Metabase will automatically choose a good way to bin your results. B
...
+
+- Set the colors and labels for the series on your chart
```
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]:
```suggestion
- Set the colors and labels for the series on your chart.
```

File: docs/users-guide/05-visualizing-results.md [hunk 4]
```diff
@@ -189,10 +189,32 @@ By default, Metabase will automatically choose a good way to bin your results. B
...
+- Change the style of your lines for Line and Area charts, and choose whether or not to display dots on them
```
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]:
```suggestion
- Change the style of your lines for Line and Area charts, and choose whether or not to display dots on them.
```

File: docs/users-guide/05-visualizing-results.md (hunk 3 above)
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]:
```suggestion
- Set the colors and labels for the series on your chart.
```

File: docs/users-guide/05-visualizing-results.md (hunk 2 above)
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]: In vs on?
```suggestion
There's quite a bit you can do in this tab:
```

File: docs/users-guide/05-visualizing-results.md (hunk 4 above)
Comment (kdoh) [inline docs/users-guide/05-visualizing-results.md]:
```suggestion
- Change the style of your lines for Line and Area charts, and choose whether or not to display dots on them.
```

**Review by kdoh — CHANGES_REQUESTED** [review]
Just a couple super tiny fixes for missing `.`s. Otherwise looks good.

File: docs/users-guide/05-visualizing-results.md [hunk 5]
```diff
@@ -189,10 +189,32 @@ By default, Metabase will automatically choose a good way to bin your results. B
...
+- Use the "Replace missing values with…" setting to change how your chart deals with missing values. You can user linear interpolation, or display those points as zero or as nothing.
```
Comment (dacort) [inline docs/users-guide/05-visualizing-results.md]:
```suggestion
- Use the "Replace missing values with…" setting to change how your chart deals with missing values. You can use linear interpolation, or display those points as zero or as nothing.
```

File: docs/users-guide/writing-sql.md [hunk 6]
```diff
@@ -16,9 +16,9 @@ You can write SQL (or your database's native querying language) directly into th
...
+To try it out, select the Sample Dataset, then type the command `select sum(subtotal), created_at from orders group by created_at`. Don't worry if you don't understand this just yet. Click the blue run button and note the table that comes back is the same as if you had asked for the sum of Subtotal in the Orders table, grouped by Created At.
 
-**Tip!** You can run your SQL query by pressing `ctrl` + `enter` on Windows, or `command` + `return` on a Mac. You can also run only a specific section of a query by selecting the part you'd like to run and pressing `ctrl` + `alt` + `enter` on Windows, or `command` + `option` + `return` on a Mac.
+**Tip:** You can run your SQL query by pressing **ctrl + enter** on Windows and Linux, or **⌘ + return** on a Mac. You can also run only the selected section of a query by highlight the part you'd like to run before clicking the run button or using the run shortcut key.
```
Comment (dacort) [inline docs/users-guide/writing-sql.md]: 🤔 Both ctrl+enter and cmd+return work on a Mac.

Comment (dacort) [inline docs/users-guide/writing-sql.md]:
```suggestion
**Tip:** You can run your SQL query by pressing **ctrl + enter** on Windows and Linux, or **⌘ + return** on a Mac. You can also run only the selected section of a query by highlighting the part you'd like to run before clicking the run button or using the run shortcut key.
```

**Review by dacort — CHANGES_REQUESTED** [review]
Couple minor changes.

**Comment by mazameli** [comment]
@kdoh @dacort Thanks for the review! Made the requested changes. Let me know if you spot anything else.

**Review by kdoh — APPROVED** [review]

**Review by dacort — APPROVED** [review]

**Pull request merged**